typescript-babel-combination/babel.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = (api) => {
+  // Cache configuration is a required option
+  api.cache(false);
+
+  const presets = [
+    "@babel/preset-typescript",
+    "@babel/preset-env"
+  ];
+  
+  const plugins = [
```

Comment:
Neither of these seem to be being used. Can be removed from here and as deps.

Can TS do ESM without Babel? I dont think it can, but if it can its probably worth us adding another JS feature which TS would need Babel for (e.g. spreading, optional chaining, null coalescing etc).

Reply:
At first, I just refer to this repo from Microsoft, (https://github.com/microsoft/TypeScript-Babel-Starter). But as you said, Babel is not used for ESM here, I've removed them. Thanks!